### PR TITLE
Update to spark to avoid security vulnerability (thank you github :) )

### PR DESCRIPTION
**Pull request by ryankilleaactian**
Update spark dependency from 1.6.1 to 2.2.0

### CODE REVIEW

File: src/main/scala/com/actian/KafkaSparkVector/SimpleApp.scala [hunk 1]
```diff
@@ -1,7 +1,7 @@
 package com.actian.KafkaSparkVector
 
 /*
- Copyright 2015 Actian Corporation
+ Copyright 2019 Actian Corporation
```
Comment (clach04) [inline src/main/scala/com/actian/KafkaSparkVector/SimpleApp.scala:4]: not a show-stopper, recommend updating this to a range, e.g. 2015-2019

File: src/main/scala/com/actian/KafkaSparkVector/SimpleApp.scala [hunk 2]
```diff
@@ -36,17 +42,13 @@ object SimpleApp {
...
-    
-    
-
-
 
     // Init SQL context
     val sqlContext = new SQLContext(sc) 
     import sqlContext.implicits._
     
     // Change connection properties to fit your demo environment
```
Comment (clach04) [inline src/main/scala/com/actian/KafkaSparkVector/SimpleApp.scala:50]: As the readme has been updated to remove the things to look at editing/customizing, recommend adding some easy search term (e.g. EDIT_ME) and mentioning in the readme too. Again not a show stopper.

**Review by clach04 — APPROVED** [review]
👍 Looks good, thanks for the PR!

File: src/main/scala/com/actian/KafkaSparkVector/SimpleApp.scala (hunk 1 above)
Comment (ryankilleaactian) [reply src/main/scala/com/actian/KafkaSparkVector/SimpleApp.scala:4]: Ok, I'll change that

File: src/main/scala/com/actian/KafkaSparkVector/SimpleApp.scala (hunk 2 above)
Comment (ryankilleaactian) [reply src/main/scala/com/actian/KafkaSparkVector/SimpleApp.scala:50]: Agreed, will change that.

**Pull request merged**